VotingApplication/VotingApplication.Web/Scripts/Services/PollService.js

```diff
@@ -1,20 +1,9 @@
 ﻿(function () {
-    angular.module('GVA.Voting').factory('PollService', ['$location', '$http', function ($location, $http) {
+    angular.module('GVA.Common').factory('PollService', ['$location', '$http', function ($location, $http) {
```

Comment:
This is rather a long line. Would be a bit easier on the eyes if .module and .controller were each on their own line.